packages/contentstack-migration/examples/05-Update-reference-entry-from-mapper.js

```diff
@@ -0,0 +1,156 @@
+const fs = require('fs');
+const chalk = require('chalk');
+const path = require('path');
+module.exports = async ({ migration, stackSDKInstance, managementAPIClient, config }) => {
+  const modules = ['entries', 'assets', 'extensions', 'marketplace_apps'];
+
+  const readAllModulesUids = (filePath) => {
+    let uidMapping = {};
+
+    modules.forEach((module) => {
+      const mappingFilePath = path.join(filePath, 'mapper', module, 'uid-mapping.json');
+      if (fs.existsSync(mappingFilePath)) {
+        const mappedIds = JSON.parse(fs.readFileSync(mappingFilePath, 'utf-8'));
+
+        if (module === 'marketplace_apps') {
+          Object.values(mappedIds).forEach((ids) => Object.assign(uidMapping, ids));
+        } else {
+          Object.assign(uidMapping, mappedIds);
+        }
+      }
+    });
+
+    return uidMapping;
+  };
+
+  const getEntries = async (ct) => {
+    try {
+      let entries = [];
+      let skip = 0;
+      let limit = 100;
+      let count = 0;
+      while (skip <= count) {
+        const res = await stackSDKInstance.contentType(ct).entry().query({ include_count: true, skip, limit }).find();
+        count = res.count;
+        skip += limit;
+        entries = [...entries, ...res.items];
```

Comment:
It seems unnecessary loop happening here `[...entries, ...res.items]`.

It can be just `entries.push(...res.items)` or `entries = entries.concat(...res.items)`

Reply:
Added